modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/catalog/resource/v1_0/test/SkuResourceTest.java

```diff
@@ -79,6 +80,25 @@ public void setUp() throws Exception {
 			10);
 	}
 
+	@Override
+	@Test
+	public void testGetChannelByExternalReferenceCodeChannelExternalReferenceCodeProductByExternalReferenceCodeProductExternalReferenceCodeSkuByExternalReferenceCodeSkuExternalReferenceCode()
```

Comment:
Could you please remember me why we have this method override and we cannot use the "base" one?

Reply:
I had to override it because the base one calls testGetChannelByExternalReferenceCodeChannelExternalReferenceCodeProductByExternalReferenceCodeProductExternalReferenceCodeSkuByExternalReferenceCodeSkuExternalReferenceCode_getSkuExternalReferenceCode() instead of postSku.getExternalReferenceCode(). This method with the long method name are parameterless, so I can't even pass sku to it thus there is no other way to make it work unless I override it.